### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aesteve-rh**
Update old repositories list and add Fedora specific packages.

### CODE REVIEW

**Review by nirs — COMMENTED** [review]
Thanks!

File: README.md [hunk 1]
```diff
@@ -32,9 +32,8 @@ date, please check the [Gerrit vdsm project](https://gerrit.ovirt.org/q/project:
 
 ## Manual installation
 
-Add ovirt repositories to your repositories list:
-
-    sudo dnf install -y http://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm
+Add ovirt repositories to your repositories list, please check
+[copr master-snapshot repositories](https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/).
```
[the remark this answers is not in the document]
Comment (aesteve-rh) [reply README.md:36]: Sure, I leave the link though, and just add the instructions for stream 8, right?

**Comment by nirs** [comment]
@aesteve-rh please also add:

    Signed-off-by: Albert Esteve <aesteve@redhat.com>

To the commit message. For this commit, the best way is:

    git commit --amend --signoff

For next commits, modify your .git/hooks/commit-msg to add this automatically:

    $ cat .git/hooks/commit-msg
    #!/bin/sh

    # Add signed-off-by lines
    SOB=$(git var GIT_AUTHOR_IDENT | sed -n 's/^\(.*>\).*$/Signed-off-by: \1/p')
    grep -qs "^$SOB" "$1" || echo "$SOB" >> "$1"

I guess this is another thing missing in the project documentation. I think this is documented
in ovirt.org somewhere, but having it in the project docs is better.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (nirs) [reply README.md:36]: yes, sounds good, maybe something like:

   For CentOS Stream 8 use:

       ...

   For more info see ...

File: README.md [hunk 2]
```diff
@@ -32,9 +32,15 @@ date, please check the [Gerrit vdsm project](https://gerrit.ovirt.org/q/project:
...
 ## Manual installation
 
-Add ovirt repositories to your repositories list:
+Add ovirt repositories to your repositories list.
 
-    sudo dnf install -y http://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm
+For CentOS Stream 8 use:
+
+    dnf copr enable -y ovirt/ovirt-master-snapshot centos-stream-8
+    dnf install -y ovirt-release-master
```
Comment (nirs) [inline README.md:40]: Old command used sudo, like rest of the file. I guess dropping sudo was not your intent?

**Comment by mz-pdm** [comment]
> For next commits, modify your .git/hooks/commit-msg to add this automatically:

In the sample default hooks created by ``git init``, at least on my system, it's recommended to use prepare-commit-msg hook  (which makes sense) to insert Signed-off-by:. It also includes (somewhat different) code that one can uncomment to insert it.

They also say that inserting Signed-off-by in hooks is not a good idea but they don't say why. I think using the hook is OK.

**Comment by nirs** [comment]
> > For next commits, modify your .git/hooks/commit-msg to add this automatically:
> 
> In the sample default hooks created by `git init`, at least on my system, it's recommended to use prepare-commit-msg hook (which makes sense) to insert Signed-off-by:. It also includes (somewhat different) code that one can uncomment to insert it.

I never tried the prepare-commit-msg hook. What is the advantage?

Using the code added by git init sounds better. The code I added came from the oVirt
commit-msg hook linked from ovirt.org.

> They also say that inserting Signed-off-by in hooks is not a good idea but they don't say why. I think using the hook is OK.

It will be best to eliminate this header, the semantics of the header are not clear
and I think we just waste time and make it harder for new contributors when we require
it.

**Review by nirs — APPROVED** [review]

**Comment by mz-pdm** [comment]
> I never tried the prepare-commit-msg hook. What is the advantage?

That you see it in the commit message already when you write it.

> It will be best to eliminate this header, the semantics of the header are not clear and I think we just waste time and make it harder for new contributors when we require it.

I agree, I doubt it serves a useful purpose nowadays. In case we need some confirmation from contributors (do we?), it's easier to have it as a part of the PR template.

**Review by mz-pdm — APPROVED** [review]

**Comment by nirs** [comment]
> > I never tried the prepare-commit-msg hook. What is the advantage?
> 
> That you see it in the commit message already when you write it.

Nicer.

> > It will be best to eliminate this header, the semantics of the header are not clear and I think we just waste time and make it harder for new contributors when we require it.
> 
> I agree, I doubt it serves a useful purpose nowadays. In case we need some confirmation from contributors (do we?), it's easier to have it as a part of the PR template.

PR template is not part of the source, not sure how useful it is.

Lets move the discussion about the "signed-of-by" to the devel mailing list.

**Comment by barpavel** [comment]
After the patch is merged, you can delete the `patch_readme` branch from both github and locally (with `git branch -D patch_readme`).

**Pull request merged**